src/main/memoryjs.d.ts

```diff
@@ -33,10 +33,33 @@ declare module 'memoryjs' {
 	export function getProcesses(processId: number, callback?: Callback<ModuleObject[]>): ModuleObject[];
 
 	// Memory
-
 	export type Vector3 = { x: number, y: number, z: number };
 	export type Vector4 = { x: number, y: number, z: number, w: number };
-	export type DataType = "byte" | "int" | "int32" | "uint32" | "int64" | "uint64" | "dword" | "short" | "long" | "float" | "double" | "bool" | "boolean" | "ptr" | "pointer" | "str" | "string" | "vec3" | "vector3" | "vec4" | "vector4";
+
+	// Data type constants
+	export enum DataType {
```

Comment:
I know a lot can change in TypeScript in 3 years, but I've always been warned away from declaring enums inside definition files.

https://lukasbehal.com/2017-05-22-enums-in-declaration-files/